src/main/java/org/sagebionetworks/web/client/widget/entity/controller/ProvenanceEditorWidgetViewImpl.java

```diff
@@ -0,0 +1,168 @@
+package org.sagebionetworks.web.client.widget.entity.controller;
+
+import org.gwtbootstrap3.client.ui.Button;
+import org.gwtbootstrap3.client.ui.Modal;
+import org.gwtbootstrap3.client.ui.TextArea;
+import org.gwtbootstrap3.client.ui.TextBox;
+import org.sagebionetworks.web.client.DisplayUtils;
+
+import com.google.gwt.event.dom.client.ClickEvent;
+import com.google.gwt.event.dom.client.ClickHandler;
+import com.google.gwt.uibinder.client.UiBinder;
+import com.google.gwt.uibinder.client.UiField;
+import com.google.gwt.user.client.ui.Composite;
+import com.google.gwt.user.client.ui.IsWidget;
+import com.google.gwt.user.client.ui.SimplePanel;
+import com.google.gwt.user.client.ui.Widget;
+import com.google.inject.Inject;
+
+public class ProvenanceEditorWidgetViewImpl extends Composite implements ProvenanceEditorWidgetView {
+
+	String PROV_LINK = "https://www.synapse.org/#!Wiki:syn2305384/ENTITY/62865";
```

Comment:
Help should go to "#!Help:Provenance", and you should add a new Help page mapping.  See SynapseClientImpl.initHelpPagesMap().  It involves adding 2 lines to the portal.properties files (to point to syn2305384 wiki id 62865).